src/mailingui/components/buttons/Buttons.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { FC, ReactNode, CSSProperties } from "react";
+import { Button as ReactEmailButton } from "./ReactEmailButtonFork";
+
+type sizes = "xs" | "sm" | "md" | "lg" | "xl" | "2xl";
+
+interface ButtonProps {
+  href: string;
+  children: ReactNode;
+  backgroundColor?: CSSProperties["color"];
+  color?: CSSProperties["color"];
+  borderColor?: CSSProperties["color"];
+  rounded?: "none" | "sm" | "md" | "lg" | "full";
```

Comment:
I'm thinking, should we make this just a plain integer? (overridable with a template with a reasonable default?)